deb/openmediavault/debian/openmediavault.postinst

```diff
@@ -75,6 +75,11 @@ case "$1" in
 					--comment 'openmediavault WebGUI Administrator' \
 					admin
 			fi
+			####################################################################
+			# User 'admin' is added to group 'openmediavault-webgui' to provide
+			# access to web interface.
+			####################################################################
+			adduser admin openmediavault-webgui
```

Comment:
This user is for PHP-FPM only.

Reply:
Indeed, but I wanted to avoid to create another group as we have already 3 new groups for OMV.